views/partials/tournament_2v2_game_finished.ejs

```diff
@@ -0,0 +1,11 @@
+<div class="rowContainer" style="grid-column: 1 / 13;">
```

Comment:
Having 4 different files to create very similar rows seems unnecessary. One ejs should be enough